demos/SSHRemoting/README.md

```diff
@@ -173,7 +173,7 @@ Microsoft Windows [Version 10.0.10586]
 #
 # Windows to Windows
 #
-C:\Users\PSUser\Documents>"C:\Program Files\PowerShell\6.0.0.17\powershell.exe"
+C:\Users\PSUser\Documents>pwsh.exe
```

Comment:
We should update the msi installer to add it to %PATH%, I wanted to remove it since it references a specific version path.  I'll do the msi update separate from this PR.